javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentArgUtil.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap;
+
+import javax.annotation.Nullable;
+
+public class AgentArgUtil {
```

Comment:
does this class need to be public? does it make sense to have a separate class for this at all?

Reply:
no - deleted the class